### PR TITLE
Add docs on how to extend the Scaffolder with Field Extensions.

### PR DESCRIPTION
**Pull request by Xantier**
https://deploy-preview-1230--roadie.netlify.app/docs/scaffolder/custom-fields/

### CODE REVIEW

File: content/docs/scaffolder/custom-fields/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: Custom Scaffolder Field Extensions
+publishedDate: '2024-02-28'
+description: How to develop and register custom Scaffolder Field Extensions 
+---
+
+You can extend and customize the available Scaffolder functionality by using [Roadie Custom Plugins](/docs/custom-plugins/overview). You can use the same workflow and development experience to build your own Custom Scaffolder Field Extensions as any other plugin, using the Roadie CLI.
+
+To see more information on how to develop custom plugins. It is recommended to navigate to the [Custom Plugins Documentation](/docs/custom-plugins/getting-started/). To construct the correct type of React components that are compatible to be registered as Scaffolder Field Extensions, the recommended approach is to [follow the tutorial on the Open Source Backstage documentation page.](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions/)
+
+
```
Comment (samnixon87) [inline content/docs/scaffolder/custom-fields/index.md:11]: `### Registering your Field Extension`

File: content/docs/scaffolder/custom-fields/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Custom Scaffolder Field Extensions
+publishedDate: '2024-02-28'
+description: How to develop and register custom Scaffolder Field Extensions 
+---
+
+You can extend and customize the available Scaffolder functionality by using [Roadie Custom Plugins](/docs/custom-plugins/overview). You can use the same workflow and development experience to build your own Custom Scaffolder Field Extensions as any other plugin, using the Roadie CLI.
+
+To see more information on how to develop custom plugins. It is recommended to navigate to the [Custom Plugins Documentation](/docs/custom-plugins/getting-started/). To construct the correct type of React components that are compatible to be registered as Scaffolder Field Extensions, the recommended approach is to [follow the tutorial on the Open Source Backstage documentation page.](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions/)
```
Comment (samnixon87) [inline content/docs/scaffolder/custom-fields/index.md:9]: `### Constructing React components for your Field Extension`

To construct the correct type of React components that are compatible to be registered as Scaffolder Field Extensions, the recommended approach is to [follow the tutorial on the Open Source Backstage documentation page.](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions/)

File: content/docs/scaffolder/custom-fields/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Custom Scaffolder Field Extensions
+publishedDate: '2024-02-28'
+description: How to develop and register custom Scaffolder Field Extensions 
+---
+
+You can extend and customize the available Scaffolder functionality by using [Roadie Custom Plugins](/docs/custom-plugins/overview). You can use the same workflow and development experience to build your own Custom Scaffolder Field Extensions as any other plugin, using the Roadie CLI.
```
Comment (samnixon87) [inline content/docs/scaffolder/custom-fields/index.md:7]: You can extend and customize the available Scaffolder functionality by using [Roadie Custom Plugins](/docs/custom-plugins/overview). You can use the same workflow and development experience to build your own Custom Scaffolder Field Extensions as any other plugin, using the Roadie CLI. To see more information on how to develop custom plugins navigate to the [Custom Plugins Documentation](/docs/custom-plugins/getting-started/).

File: content/docs/scaffolder/custom-fields/index.md (hunk 2 above)
Comment (Xantier) [reply content/docs/scaffolder/custom-fields/index.md:9]: please use the suggestion feature on GitHub for these. Much easier to handle

**Review by samnixon87 — APPROVED** [review]

**Pull request merged**